docs/README.md

```diff
@@ -0,0 +1,38 @@
+<p align="center">
+  <a href="#">
+    <img alt="Ionic Logo" src="https://github.com/ionic-team/ionic-framework/blob/main/.github/assets/logo.png?raw=true" width="60" />
+  </a>
+</p>
+
+<h1 align="center">
+  Ionic Framework Developer Resources
+</h1>
+
+<p align="center">
+  This documentation includes guidelines on contributing, coding conventions, best practices, testing steps, and more. It should serve as a collective resource for all documentation related to developing Ionic Framework.
+</p>
+
+## What is Ionic Framework?
+
+Ionic Framework is an open source app development toolkit for building modern, fast, top-quality cross-platform native and Progressive Web Apps from a single codebase with JavaScript and the Web.
+
+It is based on <a href="https://www.webcomponents.org/introduction">Web Components</a>, which enables significant performance, usability, and feature improvements alongside support for popular web frameworks like <a href="https://angular.io/">Angular</a>, <a href="https://reactjs.com/">React</a>, and <a href="https://vuejs.org/">Vue</a>.
+
+## Guides
+
+| Guide                                   | Description                                                                              |
+| ----------------------------------------| ---------------------------------------------------------------------------------------- |
+| [Contributing](./CONTRIBUTING.md)       | How to contribute including creating pull requests, commit message guidelines, and more. |
+| [Component Guide](./component-guide.md) | Guidelines for implementing component states, accessibility, and more.                   |
```

Comment:
```suggestion
| [Component Guide](./COMPONENT-GUIDE.md) | Guidelines for implementing component states, accessibility, and more.                   |
```
Else it gives a 404.

Reply:
Good catch. I had lowercased the file locally but I had to run `git mv` to make git recognize it. It is fixed now: https://github.com/ionic-team/ionic-framework/pull/29266/commits/216acf9be322810cd4da50f987391d5fea7d99d6